wrappers/opencv/face/rs-face-cv.cpp

```diff
@@ -0,0 +1,65 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2017 Intel Corporation. All Rights Reserved.
```

Comment:
2019

Reply:
This file needs to be taken out of the changes -- the opencv/face/ example is not going in...